tests_fmt/test.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
```

Comment:
There is no need for this to be in a subdirectory. Put it in `tools` along with `testparser.sh`.

Reply:
We moved it to `tools` and called it `testformatter.sh`